hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -232,7 +243,16 @@ public int read(String table, String key, Set<String> fields, HashMap<String,Byt
                     g.addColumn(_columnFamilyBytes, Bytes.toBytes(field));
                 }
             }
-            r = _table.get(g);
+            if (multiGetsCount > 0) {
+              multiGets.add(g);
+              if (multiGetsCount == multiGets.size()) {
+                r = _table.get(multiGets);
```

Comment:
what happens if one of the multigets fail? do we just fail all of them?

for those gets that get batched, we're just returning 'Ok' preemptively, right?

won't this batching up distort the latency distribution?